example/index.tsx

```diff
@@ -57,6 +60,9 @@ const App = () => {
               proxyUrl={pdfProxyUrl}
             />
           </Route>
+          <Route path="/axisnow-encrypted">
```

Comment:
Only if this is easy:  Can we get this to only show up if you actually have the axis stuff?  (Or throw a reasonable error message if you don't have it)?

Reply:
Did that here: https://github.com/NYPL-Simplified/web-reader/pull/43